src/server/system_services/pool_server.js

```diff
@@ -1010,6 +1013,49 @@ function get_namespace_resource_info(namespace_resource) {
     return info;
 }
 
+function get_namespace_resource_operator_info(req) {
+    if (req.auth.role !== 'operator') {
+        throw new RpcError('UNAUTHORIZED', 'Must be operator');
+    }
+    const namespace_resource = find_namespace_resource_by_name(req);
+    if (!namespace_resource.namespace_store ||
+        !namespace_resource.namespace_store.need_k8s_sync) return {};
+
+    const info = _.omitBy({
+        access_key: namespace_resource.connection.access_key,
+        secret_key: namespace_resource.connection.secret_key,
+        need_k8s_sync: namespace_resource.namespace_store.need_k8s_sync
+    }, _.isUndefined);
+
+    return info;
+}
+
+function unset_namespace_store_sync(req) {
```

Comment:
shouldn't this be called from `create_namespace_resource`?

Reply:
we are not calling ```create_namespace_resource``` from the operator side since we already know it exist,
I had to add this api because I saw we eventually are not calling it..